src/AutoRest.CSharp/Common/Input/CodeModelPartials.cs

```diff
@@ -133,6 +133,13 @@ public string[] Formats
         public string? Format => TryGetValue("x-ms-format", out object? value) ? value?.ToString() : null;
     }
 
+    internal static class XMsFormat
```

Comment:
Should this be `XmsFormat`?
```suggestion
    internal static class XmsFormat
```

Reply:
This comes from x-ms-format, so I use the camel case XMsFormat.